### ISSUE TITLE
Can't detect open windows when running on boot

### ISSUE THREAD

**Issue by danysvirsky** [microsoft/WinAppDriver#1200]
I'm trying to start WinAppDriver on boot on AWS EC2 using user-data script.
`Start-Process "C:\Program Files (x86)\Windows Application Driver\WInAppDriver.exe -ArgumentList $HOST_IP 4732`

The process starts and I'm able to connect to it remotely but if I try to interact with an application I get an error that no open windows were detected. If I kill the process started on boot and run the same command manually, everything works.

**Comment by mohideenmj** [comment]
Yes me also facing the same issue. While trying to run the process through Jenkins file, it starts winappdriver but tests are not able to find the application which results in an error( WebDriverException: Message: Failed to locate opened application window with appId:). Both application and winappdriver.exe are running as a background process
